app/routes/report/variants.js

```diff
@@ -38,6 +38,8 @@ const therapeuticAssociationFilter = {
     {[Op.is]: literal('distinct from \'msi\'')},
     {[Op.is]: literal('distinct from \'tmb\'')},
   ]},
+  // Regex filter for finding columns with 2 or more spaces (excludes generic "geneX mutation")
+  kbVariant: {[Op.regexp]: '^(.*\\s.*){2,}$'},
```

Comment:
should also check that 'mutation' is in the string somewhere to make sure other types of variants are not filtered out by this

Reply:
actually the check will also need to not exclude " high mutation burden high signature"